README.md

```diff
@@ -19,6 +19,24 @@ The idea is to place the proxy between a client (e.g. a frontend app) and a web
 </p>
 
 
+## FAQ
+
+<details>
+  <summary>Can I use this in production?</summary>
+  This tool was originaly meant for development mode. You can use this in production but we cannot give you any promise of security. Also running the jsonschema-validation is very CPU-expensive and you likely do not want to validate in both directions in production because of that overhead.
```

Comment:
```suggestion
  This tool was originally meant for development scenarios. You can use this in production but we cannot give you any security guarantees. Also running JSON schema validation is quite CPU-expensive and you likely do not want to validate in both directions in production because of that overhead.
```